domains/innocentzero.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "personal blog - innocentzero",
+  "repo": "https://codeberg.org/innocent_zero/pages",
+  "owner": {
+    "username": "innocent_zero",
```

this should be your github username
```suggestion
    "username": "InnocentZero",
```